runtime/environment.go

```diff
@@ -63,18 +63,23 @@ type Environment interface {
 }
 
 type interpreterEnvironment struct {
-	config                                Config
-	baseActivation                        *interpreter.VariableActivation
-	baseValueActivation                   *sema.VariableActivation
-	runtimeInterface                      Interface
-	storage                               *Storage
-	coverageReport                        *CoverageReport
-	codesAndPrograms                      codesAndPrograms
+	config Config
+
```

Comment:
Can we remove the newlines here so the fields are all vertically aligned?

Reply:
Hmm, intentionally added newlines to group related fields and hopped it would help

Reply:
I see. In that case I suppose it's fine but it is frustrating that grouping the fields this way also ruins the alignment.

Reply:
yeah, I usually have whitespace changes disabled on GitHub for that reason